CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.22)
 cmake_policy(SET CMP0091 NEW)
 cmake_policy(SET CMP0067 NEW)
 
-set(Bun_VERSION "1.1.0")
+set(Bun_VERSION "1.1.1")
```

Comment:
I've seen a few diff v1_1_1 things or was that a 1_1_0 thing. This should be in a sep PR though as it's versioning related.